helm/observability-operator/files/alertmanager/alertmanager.yaml.helm-template

```diff
@@ -0,0 +1,504 @@
+global:
+  resolve_timeout: 5m
+{{- if .alertmanager.proxyURL }}
+  http_config:
+    proxy_url:  {{ .alertmanager.proxyURL }}
+{{- end }}
+{{- if .alertmanager.slackAPIToken }}
```

Comment:
Is there a CAPA installation where we do not use slackAPIToken?

Reply:
I think not, but it doesn't hurt to keep it for now, I'd rather be safe.